--- /home/dazong/Desktop/final-project-dazong-chen/final_buildroot/buildroot/output/build/linux-custom/arch/arm/boot/dts/am335x-bone-common.dtsi	2019-11-04 11:36:08.000000000 -0700
+++ /home/dazong/Desktop/final-project-dazong-chen/am335x-bone-common.dtsi	2021-11-14 10:11:36.954383601 -0700
@@ -79,12 +79,19 @@
 		>;
 	};
 
-//	i2c2_pins: pinmux_i2c2_pins {
-//		pinctrl-single,pins = <
-//			AM33XX_IOPAD(0x978, PIN_INPUT_PULLUP | MUX_MODE3)	/* uart1_ctsn.i2c2_sda */
-//			AM33XX_IOPAD(0x97c, PIN_INPUT_PULLUP | MUX_MODE3)	/* uart1_rtsn.i2c2_scl */
-//		>;
-//	};
+	i2c1_pins: pinmux_i2c1_pins {
+		pinctrl-single,pins = <
+			AM33XX_IOPAD(0x958, PIN_INPUT_PULLUP | MUX_MODE2)       /* spi0_d1.i2c1_sda */
+			AM33XX_IOPAD(0x95c, PIN_INPUT_PULLUP | MUX_MODE2)       /* spi0_cs0.i2c1_scl */
+		>;
+	};
+
+	i2c2_pins: pinmux_i2c2_pins {
+		pinctrl-single,pins = <
+			AM33XX_IOPAD(0x978, PIN_INPUT_PULLUP | MUX_MODE3)	/* uart1_ctsn.i2c2_sda */
+			AM33XX_IOPAD(0x97c, PIN_INPUT_PULLUP | MUX_MODE3)	/* uart1_rtsn.i2c2_scl */
+		>;
+	};
 
 	uart0_pins: pinmux_uart0_pins {
 		pinctrl-single,pins = <
@@ -285,6 +292,14 @@
 	};
 };
 
+&i2c1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&i2c1_pins>;
+
+	status = "okay";
+	clock-frequency = <100000>;
+};
+
 &i2c2 {
 	pinctrl-names = "default";
 	//pinctrl-0 = <&i2c2_pins>;

